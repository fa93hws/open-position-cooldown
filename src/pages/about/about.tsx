import * as React from 'react';
import { Heading, Box, Text } from 'rebass';

import { Footer } from './footer/footer';

export const AboutPage = () => (
  <Box p={[4, 5]} as="article">
    <Heading fontWeight="heading" fontSize={[4, 5]} variant="h1">
      APP的目的:
    </Heading>
    <Text fontSize={[2, 3]} mt={[1, 2]} as="p">
      离婚居然要冷静期，买股票居然不用？真是不可思议。
    </Text>
    <Text fontSize={[2, 3]} mt={[1, 2]} as="p">
      买股票前先想想好吧，不猜涨跌，只应对。
    </Text>
    <Footer />
  </Box>
);
