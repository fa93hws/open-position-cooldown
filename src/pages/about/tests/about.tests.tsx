import * as React from 'react';
import { AboutPage } from '../about';

describe('AboutPage', () => {
  it('renders it', () => {
    expect(<AboutPage />).toMatchRenderedSnapshot();
  });
});
