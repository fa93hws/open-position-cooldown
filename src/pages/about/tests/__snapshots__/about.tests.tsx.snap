// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutPage renders it 1`] = `
<article
  class="css-b6tmei"
>
  <h2
    class="css-121t47l"
  >
    APP的目的:
  </h2>
  <p
    class="css-14ksygj"
  >
    离婚居然要冷静期，买股票居然不用？真是不可思议。
  </p>
  <p
    class="css-14ksygj"
  >
    买股票前先想想好吧，不猜涨跌，只应对。
  </p>
  <div
    class="MuiBox-root MuiBox-root-5 makeStyles-container-1"
  >
    <div
      class="MuiContainer-root makeStyles-linksContainer-2 MuiContainer-maxWidthLg"
    >
      <a
        class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
        href="https://github.com/fa93hws/"
        target="__blank"
      >
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root makeStyles-icon-3"
          focusable="false"
          style="padding:4px"
          viewBox="0 0 24 24"
        >
          <path
            d="M12 .3a12 12 0 0 0-3.8 23.4c.6.1.8-.3.8-.6v-2c-3.3.7-4-1.6-4-1.6-.6-1.4-1.4-1.8-1.4-1.8-1-.7.1-.7.1-.7 1.2 0 1.9 1.2 1.9 1.2 1 1.8 2.8 1.3 3.5 1 0-.8.4-1.3.7-1.6-2.7-.3-5.5-1.3-5.5-6 0-1.2.5-2.3 1.3-3.1-.2-.4-.6-1.6 0-3.2 0 0 1-.3 3.4 1.2a11.5 11.5 0 0 1 6 0c2.3-1.5 3.3-1.2 3.3-1.2.6 1.6.2 2.8 0 3.2.9.8 1.3 1.9 1.3 3.2 0 4.6-2.8 5.6-5.5 5.9.5.4.9 1 .9 2.2v3.3c0 .3.1.7.8.6A12 12 0 0 0 12 .3"
          />
        </svg>
      </a>
      <a
        class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
        href="https://www.weibo.com/hinanawi/"
        target="__blank"
      >
        <div
          class="MuiBox-root MuiBox-root-6 makeStyles-customIconWrapper-4"
        >
          <i />
        </div>
      </a>
      <a
        class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
        href="https://github.com/fa93hws/open-position-cooldown"
        target="__blank"
      >
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root makeStyles-icon-3"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M9.4 16.6L4.8 12l4.6-4.6L8 6l-6 6 6 6 1.4-1.4zm5.2 0l4.6-4.6-4.6-4.6L16 6l6 6-6 6-1.4-1.4z"
          />
        </svg>
      </a>
    </div>
  </div>
</article>
`;
